__tests__/caesar_cipher.test.js

```diff
@@ -0,0 +1,22 @@
+'use strick'
+import { wrap } from '../src/caesar_cipher'
+import { cipher } from '../src/caesar_cipher'
+
+
+
+describe('Ceaser Cipher Tests', () => {
+  
+  test('word transposition same case', () => {
```

Comment:
This test should be this way, `caesarCipher('Microverse', 7)).toBe('Tpjyvclyzl')`